cmd/subnetcmd/describe.go

```diff
@@ -227,6 +229,27 @@ func printPrecompileTable(genesis core.Genesis) {
 	}
 }
 
+func buildAddressTable(
```

Comment:
Can you rewrite this method so it returns a table and doesn't have side effects?

Reply:
renamed func to `appendToAddressTable`